packages/wallet-adapter-core/package.json

```
@@ -52,11 +52,11 @@
   "dependencies": {
     "@aptos-connect/wallet-adapter-plugin": "^1.0.1",
     "@aptos-labs/wallet-standard": "^0.1.0",
-    "@atomrigslab/aptos-wallet-adapter": "^0.1.20",
+    "@atomrigslab/aptos-wallet-adapter": "^0.1.21",
     "@mizuwallet-sdk/aptos-wallet-adapter": "^0.1.5",
     "buffer": "^6.0.3",
     "eventemitter3": "^4.0.7",
-    "tweetnacl": "^1.0.3"
+    "tweetnacl": "^1.0.3",
```

build is failing because of the `,` at the end - need to remove it
```suggestion
    "tweetnacl": "^1.0.3"
```